### PR TITLE
k8s-cdk conjure-up install steps update

### PR DESCRIPTION
**Pull request by battlemidget**
Update install instructions for conjure-up

### CODE REVIEW

File: fragments/k8s/cdk/README.md [hunk 1]
```diff
@@ -24,10 +24,7 @@ knowledge. It is comprised of the following components and features:
 
 Installation has been automated via [conjure-up](http://conjure-up.io/):
 
-    sudo apt-add-repository ppa:juju/stable
-    sudo apt-add-repository ppa:conjure-up/next
-    sudo apt update
-    sudo apt install conjure-up
+    sudo snap install conjure-up --classic
```
Comment (marcoceppi) [inline fragments/k8s/cdk/README.md]: Is the stable channel updated enough?

Comment (battlemidget) [inline fragments/k8s/cdk/README.md]: It uses Juju 2.0.3 stable, only edge is using rc1 and beta using juju 2.1 beta4

Comment (marcoceppi) [inline fragments/k8s/cdk/README.md]: but it has the most recent stable releases of conjure-up in the stable channel?

**Comment by chuckbutler** [comment]
Shouldn't this also be applied to the kubernetes-core readme?

File: fragments/k8s/cdk/README.md (hunk 1 above)
Comment (battlemidget) [inline fragments/k8s/cdk/README.md]: Yea this has the latest bits

Comment (battlemidget) [inline fragments/k8s/cdk/README.md]: @marcoceppi can you test the snap install on your system to make sure the failed lxd bridge doesn't interfere?

**Comment by chuckbutler** [comment]
This is lingering, and its just a README update. I think we're +1 to merge this and close the loop here.

**Pull request merged**